GPT/gpt.talon

```diff
@@ -21,6 +21,13 @@ model <user.modelPrompt> [this] (then | and) select [it]$:
     result = user.gpt_apply_prompt(modelPrompt, text)
     user.paste_and_select(result)
 
+# Runs a model prompt on the selected text and appends the result to the next line
+model <user.modelPrompt> [this] (then | and) append [it]$:
```

Comment:
I'm not sure about the spoken form here. For one I am skeptical of all the optional words. We originally went this route in cursorless and then shortened things dramatically to one consistent grammar. I am of the mind that adding support for lots of optional words is pretending we understand English, which will lead to problems long term, given that we don't

But even accepting the optional words (given that is somewhat consistent with other commands in this file), I don't think the `then | and` makes sense here, because to me it implies the selected text will get updated and then there will be an append, similar to how the command above modifies selected text and then reselects it

Maybe something like
 
```suggestion
model <user.modelPrompt> after$:
```

alternatives to `after`:

- `down`
- `append`
- `clone`

we could consider `model append <prompt>` as well

Reply:
Would be helpful to see a couple example prompts where that is awkward